universal_transfer_operator/example_dags/example_universal_transfer_operator.py

```diff
@@ -30,6 +30,22 @@
         ),
     )
 
+    transfer_non_native_s3_to_sqlite = UniversalTransferOperator(
+        task_id="transfer_non_native_s3_to_sqlite",
+        source_dataset=File(
+            path="s3://astro-sdk-test/uto/csv_files/", conn_id="aws_default", filetype=FileType.CSV
```

Comment:
Having a configurable variable/constant for the bucket name would be great, so non-Astronomer developers can also run the test suite.

We followed this pattern for the Python SDK:
https://github.com/astronomer/astro-sdk/blob/main/python-sdk/example_dags/example_amazon_s3_postgres.py#L11

Reply:
Done.